Tests/Test Helpers/TestImage.cs

```diff
@@ -0,0 +1,13 @@
+﻿using UnityEngine.UI;
+
+// if you create a custom class in a file that doesn't match the class name,
+// after the tests run Unity will complain that the compiler that imported 
+// "TestImage" isn't available anymore, so they go in their own files
+namespace UnityEditor.Experimental.EditorVR.Tests.TestHelpers
+{
+    public class TestImage : Graphic
```

Comment:
Ah, so maybe the comment is just out of date? It seems to be talking about having a class name that is different from the filename. At the very least, it doesn't seem to say the same thing you just wrote. I'd just get rid of that comment, or replace it with the thing you just replied with.

I had assumed because of the comment that this was trying to test a case where the class name is different from the filename.